src/symbolize/gimli.rs

```diff
@@ -487,7 +518,7 @@ impl Cache {
                 if !lib.segments.iter().any(|s| {
                     let svma = s.stated_virtual_memory_address;
                     let start = svma.wrapping_add(lib.bias);
-                    let end = start.wrapping_add(s.len);
+                    let end = start.saturating_add(s.len);
```

Comment:
How come this changed from wrapping to saturating?

Reply:
This calculation would overflow before due to the length being `usize::max_value()`.

Reply:
I believe this is specific to the new target added, probably from the usage of the maximum value as you mentioned. I think that should be adjusted  where the segment shouldn't have a maximum value if a bias is also set, in that case the length of the segment should be adjusted to be smaller or account for the bias.